tests/sqs_service.py

```diff
@@ -62,4 +67,40 @@ def get_tokens_metadata(self):
         
         self.tokens_metadata = response.json()
 
-        return self.tokens_metadata
\ No newline at end of file
+        return self.tokens_metadata
+
+    # Given the base and quote denom, call the SQS API endpoint /tokens/prices
+    # and return the token price in the response
+    def get_tokens_prices(self, denom, quote_denom, human_denoms="false"):
+        # Set the query parameters
+        params = {
+            "base": denom,
```

Comment:
We have support for multiple base denoms (and will have support for multiple quotes in the future). Could we expand the API to support multiple tokens?

And also add test cases with multiple denoms submitted at once

Reply:
Good idea. Done.